src/main/_docs/use-che-as-an-ide/ide-debug.md

```diff
@@ -71,3 +71,116 @@ It is also possible to add this command as a Dockerfile instruction for a custom
+
+```
+http://localhost:32810/web-php-simple/index.php?start_debug=1&debug_host=localhost&debug_port=10137
+```
+
+### Using the zDebug Extension for Chrome
+
+The [zDebug](https://chrome.google.com/webstore/detail/zdebug/gknbnafalimbhgkmichoadhmkaoingil) extension can be used for easier triggering of debug sessions from the Chrome browser. The [Zend Debugger Extension](https://chrome.google.com/webstore/detail/zend-debugger-extension/aonajadpeeaijblinaeohfdmbgdpibba) is another extension that does the same job.
+
+It is important to configure the Chrome extension properly before using it for debugging PHP apps running in a Che workspace:
+
+1. Set `Debug Host` to `localhost` or `127.0.0.1`.
+1. Set `Debug Port` to `10137`.
+1. Set `Debug Local Copy` to `No`.
+
+Note that it is not the browser that opens the debug session to the Zend Debugger Client. This is done by the PHP engine that runs in the Che workspace. The browser just tells the PHP engine to do so. So the above settings are for the PHP engine (the Zend Debugger module in particular). Thus the `Debug Host` must be set to `localhost` and not the public host of the docker container running the Che workspace.
+
+In the end the zDebug settings should look like this:
+
+![zdebug-settings.png]({{ base }}/docs/assets/imgs/zdebug-settings.png)
+
+Now you are ready to trigger the debug session:
+
+1. Open the web page to debug.
+1. Click on the zDebug toolbar button.
+1. Click on `This Page`.
+
+### Using the Zend Debugger Toolbar for Firefox
+
+The [Zend Debugger Toolbar for Firefox](https://addons.mozilla.org/firefox/addon/zend-debugger-toolbar/) can be used for easier triggering of debug sessions from the Firefox browser.
+
+After installing it, go to `Extra Stuff > Setttings` to configure the toolbar:
+
+1. Disable the `Debug Local Copy` option.
+1. Switch the `Client/IDE Settings` to `Manual Settings`.
+1. Set `Debug Port` to `10137`.
+1. Set `IP Address` to `127.0.0.1`.
+
+In the end the toolbar settings should look like this:
+
+![zend-debugger-firefox-settings.png]({{ base }}/docs/assets/imgs/zend-debugger-firefox-settings.png)
+
+Now you are ready to trigger the debug session:
+
+1. Open the web page to debug.
+1. Click on the `Debug` toolbar button.
+
+### Using Z-Ray
+
+[Z-Ray](http://www.zend.com/en/products/server/z-ray) is a productivity tool, part of [Zend Server](http://www.zend.com/en/products/zend_server), that is available in the Zend stack. Z-Ray requires no installation or configuraton. It is injected into the response coming from your PHP app and shown right in the browser you are using for development. 
+
+Among other features, it also has the capability to trigger a debug session:
+
+1. Click on the "bug" button.
+1. Click on `Debug current page`.
+
+![z-ray-debug.png]({{ base }}/docs/assets/imgs/z-ray-debug.png)
```

Comment:
Should change to ![z-ray-debug.png]({{ base }}{{site.links["z-ray-debug.png"]}})